app/src/rgb_underglow.c

```diff
@@ -335,7 +335,7 @@ int zmk_rgb_underglow_toggle() {
 }
 
 int zmk_rgb_underglow_set_hsb(struct zmk_led_hsb color) {
-    if (color.h > HUE_MAX || color.s > SAT_MAX || color.b > BRT_MAX) {
+    if (color.h > HUE_MAX || color.s > SAT_MAX || color.b > CONFIG_ZMK_RGB_UNDERGLOW_BRT_MAX) {
```

Comment:
I'm wondering if `CONFIG_ZMK_RGB_UNDERGLOW_BRT_MAX` should be used as a cap here rather than an actual hard limit. For example, if someone has a key that sets the HSB to 0, 100, 100, should that be 100% of the cap, or should it really just error?

Reply:
100% of `CONFIG_ZMK_RGB_UNDERGLOW_BRT_MAX`, I see that as a hardware limitation implemented in software.

You can also draw a paralell to dimmers in houses, they have an "installation setting" MIN and MAX that depends on the installed lightning.
From the user side a dimmer value of 100% = the MAX and an user value of 1% = MIN  (0% is off)

Reply:
I feel like that scenario should be impossible. I proposed this change with hardware limitations in mind rather than a preference or something.

Reply:
This scenario is definitely possible. All someone needs to do is add a `&rgb_ug RGB_COLOR_HSB(100, 100, 100)` and now that key silently fails to do anything because of this check.